src/schematools/types.py

```diff
@@ -536,43 +564,45 @@ def identifier(self) -> List[str]:
         # Convert identifier to a list, to be backwards compatible with older schemas
         if not isinstance(identifier, list):
             identifier = [identifier]
-        return identifier
+        return list(identifier)  # mypy pleaser
```

Comment:
Amazed this is needed to please MyPy. What version are you running?

And if it is really needed a `cast(List[str], ....)` would arguably be more appropriate.

Reply:
My mypy version is 0.812 (feb 19, 2021), seems quite recent. I agree, cast is better.